content/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lifecycle badges
+---
+
+![](/images/lifecycle/lifecycle.png)
+
+### Experimental ![](https://img.shields.io/badge/lifecycle-experimental-orange.svg) {#experimental}
+
+Very early days, a lot of churn in search of a good API.  Not on CRAN. Might not go anywhere. Use with care.
+
+### Maturing ![](https://img.shields.io/badge/lifecycle-maturing-blue.svg) {#maturing}
+
+API roughed out, but finer details likely to change. Will strive to maintain backward compatibility, but need wider usage in order to get more feedback.
+
+### Stable ![](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg) {#stable}
+
+We're happy with the API, and the unlikely to be major changes. Backward incompatible changes will only be made if absolutely critical, and will be accompanied by a change in the major version.
+
+### Retired ![](https://img.shields.io/badge/lifecycle-retired-orange.svg) {#retired}
+
+Known better replacement available elsewhere. Will remain  available on CRAN.
+
+### Archived ![](https://img.shields.io/badge/lifecycle-archived-red.svg) {#archived}
+
+Development complete. Archived on CRAN and on GitHub.
+
+### Dormant ![](https://img.shields.io/badge/lifecycle-dormant-blue.svg) {#dormant}
+
+Not currently working on it, but plan to come back to in the future.
```

Comment:
A dormant package is not completed, but is not currently under active development. We plan to return to it in the future.